### PR TITLE
Use forms for issue templates

### PR DESCRIPTION
**Pull request by Jacalz**
### Description:

GitHub now supports using forms for the issue templates. This should help make it easier for the users to fill them out with the relevant information (or more likely harder to not fill it out correctly).

As this is about issue templates, the PR targets the `master` branch.

To see the results of this, see links below:
- [BUG_TEMPLATE](https://github.com/Jacalz/fyne/blob/issues/.github/ISSUE_TEMPLATE/bug_report.yml)
- [FEATURE_TEMPLATE](https://github.com/Jacalz/fyne/blob/issues/.github/ISSUE_TEMPLATE/feature_request.yml)

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

**Comment by andydotxyz** [comment]
Nice new feature - but the 3 items at the top of the bug template seem a little much - especially to confirm code of conduct for every new issue opened.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- type: checkboxes
+  attributes:
+    label: Checklist
+    description: Please make sure you have read the following.
+    options:
+      - label: I agree to follow the [Code of Conduct](https://github.com/fyne-io/fyne/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
+        required: true
+      - label: I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
+        required: true
+      - label: This issue only relates to a single feature. I will open new issues for any other suggestions.
```
Comment (andydotxyz) [inline .github/ISSUE_TEMPLATE/feature_request.yml:13]: This should probably refer to features :)

**Review by andydotxyz — CHANGES_REQUESTED** [review]
I wonder if there is a way to make the top block easier to approve, or simpler to describe so it's not so much of a hurdle for people opening tickets.

**Comment by Jacalz** [comment]
Good call. I removed the first check entirely and changed "suggestions" to "features".

**Review by andydotxyz — APPROVED** [review]

**Pull request merged**